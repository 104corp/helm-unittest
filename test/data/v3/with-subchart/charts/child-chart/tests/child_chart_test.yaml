suite: test child_chart name
templates:
  - templates/child-chart.yaml
tests:
  - it: should succeed even with the same names
    asserts:
      - matchSnapshot: {}
