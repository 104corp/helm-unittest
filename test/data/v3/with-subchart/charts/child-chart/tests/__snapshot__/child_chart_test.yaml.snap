should succeed even with the same names:
  1: |
    apiVersion: example
    kind: ChildChart
